R/PrestoConnection.R

```diff
@@ -25,7 +25,8 @@ setClass('PrestoConnection',
     'use.trino.headers'='logical',
     'Id'='character',
     'session'='PrestoSession',
-    'extra.credentials'='character'
+    'extra.credentials'='character',
+    'bigint'='character'
```

Comment:
I think this is an ambiguous name. Maybe `bigint.class` or something would be better?

Reply:
Makes sense. Leave as is then.